app/frontend/actions.js

```diff
@@ -26,6 +26,8 @@ const submitEmailRaw = require('./helpers/submitEmailRaw')
 const captureBySentry = require('./helpers/captureBySentry')
 const submitFeedbackToSentry = require('./helpers/submitFeedbackToSentry')
 
+const adalitePoolAddress = 'adalite-pool-address'
```

Comment:
you can prepare this for later by putting it in your .env file, check how it's loaded from it in loadConfig.js

Reply:
Done.